example/lib/main.dart

```diff
@@ -33,7 +32,7 @@ You will use the Amplify CLI tool for that.  Please read the
 README.md contained within the root of this project to learn
 about what you'll need to do.
  */
-import 'amplifyconfiguration.dart';
+// import 'amplifyconfiguration.dart';
```

Comment:
Let's keep this as is. As the comment above notes, to run the example, a configuration file is needed. Commenting this out will not remove need to generate a config file, so it really just introduces an extra step to uncomment this.
```suggestion
import 'amplifyconfiguration.dart';
```